test/metabase/driver/clickhouse_test.clj

```diff
@@ -175,3 +176,17 @@
    (is (= nil (#'clickhouse-qp/extract-datetime-timezone "datetime")))
    (is (= nil (#'clickhouse-qp/extract-datetime-timezone "datetime64")))
    (is (= nil (#'clickhouse-qp/extract-datetime-timezone "datetime64(3)")))))
+
+(deftest clickhouse-insert
```

Comment:
I recommend either marking this test as `:synchronized`, just to be explicit.

Reply:
Fixed 89cdcaeafa265eb259be67f92171e006fadaef5e